engine/includes/resources/prefab.h

```diff
@@ -3,7 +3,12 @@
 
 #include <resource.h>
 
+// When building with Clang on FreeBSD, this is a failure point if not set correctly
+#if defined(__FreeBSD__)
```

Comment:
This could brake build procedure from SDK because "components" directory doesn't exist (all in engine dir)
I guess it could be solved with additional include dir in build scripts

Reply:
![Actor](https://github.com/thunder-engine/thunder/assets/59133880/87afa6d8-5be7-46fb-a261-ea17ab5814bf)

I'll need to mess with it some, but the issue is not prefab.h itself but how rendergl was linked.

Reply:
Should be good now. I also removed the few `#include "engine/includes/gui` I came across. There probably are others but it's compiling like it should now otherwise.